docs/reference/query-dsl/simple-query-string-query.asciidoc

```diff
@@ -273,33 +274,39 @@ GET /_search
 <1> The `subject` field is three times as important as the `message` field.
 
 [[simple-query-string-synonyms]]
-===== Synonyms
+===== Multi-position tokens
 
-The `simple_query_string` query supports multi-terms synonym expansion with the <<analysis-synonym-graph-tokenfilter,
-synonym_graph>> token filter. When this filter is used, the parser creates a phrase query for each multi-terms synonyms.
-For example, the following synonym: `"ny, new york"` would produce:
+The `simple_query_string` query supports
+<<token-graphs-multi-position-tokens,multi-position tokens>> produced by graph
+token filters, such as <<analysis-synonym-graph-tokenfilter,`synonym_graph`>> or
+<<analysis-word-delimiter-graph-tokenfilter,`word_delimiter_graph`>>.
```

Comment:
Maybe add some information about decompounders such as nori as well?

Reply:
Thanks @imotov. After looking at this again, I decided to remove the graph token filters reference. It's a bit redundant with the multi-position token link. I added a reference to the `nori_tokenizer` to those docs with 1cc2ab7310ce475ed292fb1a6f178986625fb2ed.